test/doctrines/Monoidal.jl

```diff
@@ -134,12 +134,6 @@ f, g = Hom(:f, A, B), Hom(:g, B, A)
 @test dom(mate(f)) == dual(B)
 @test codom(mate(f)) == dual(A)
 
-f = Hom(:f, otimes(A,B), C)
```

Comment:
With appropriate term constructors added, some version of these tests should still pass.

Reply:
fixed